client/landing/gutenboarding/onboarding-block/vertical-select/index.tsx

```diff
@@ -128,11 +124,10 @@ const VerticalSelect: FunctionComponent< InjectedStepProps > = ( {
 					className={ inputClass }
 					placeholder={ NO__( 'enter a topic' ) }
 					onChange={ handleSuggestionChangeEvent }
-					onFocus={ showSuggestions }
-					onBlur={ hideSuggestions }
+					onBlur={ () => handleSelect() }
```

Comment:
Somewhat related to [this comment](https://github.com/Automattic/wp-calypso/pull/37876/files#r350042749): How about using a dedicated method here rather than `handleSelect`? We already know that we're not passing a vertical here, so we might as well call the vertical reset pretty much directly, and drop the corresponding logic from `handleSelect`, no?